npc/jobs/2-2/crusader.txt

```diff
@@ -223,6 +209,27 @@ prt_castle,45,169,5	script	Senior Crusader	4_M_CRU_OLD,{
 	else if(CRUS_Q >= 1 && CRUS_Q <= 3) {
 		mes "Have you proven your determination with the task I have given you, or do you possess the items proving that you have received your calling?";
 		next;
+		if (countitem(Patriotism_Marks) >= 1 && countitem(Sacred_Marks) >= 1) {
+			mes "Ah...";
+			mes "I see that you have been called to become a Crusader. We are assured of your will, but now we must test your capabilities.";
+			next;
+			mes "[Michael Halig]";
+			mes "Meet with Moorenak Miyol who is training in the underground dungeon of the Prontera Castle. Go, and speak with him first.";
+			next;
+			delitem Patriotism_Marks, 1;
+			delitem Sacred_Marks, 1;
+			CRUS_Q = 4;
+			if (questprogress(3006)) {
+				changequest 3006,3009;
+			} else if (questprogress(3007)) {
+				changequest 3007,3009;
+			} else {
+				changequest 3008,3009;
```

Comment:
That's now what I mean, my point is in general about the usage of changequest rather than completequest. 
Changequest erases the questprogress back to 0 while completequest removes it from quest window and sets it to 2.
So with changequest you have no future chance to check if quest was done or not. This is not required here but in other usages it is. Changequest actually makes only sense when using for some kind of cooldown quest